app/elements/common/coduno-iron-ajax.html

```diff
@@ -45,30 +43,34 @@
 					type: String,
 					notify: true,
 					value: 'json'
-				},
-				headers: {
-					type: Object,
-					notify: true,
-					value: {}
-				},
-				url: {
-					type: String,
-					notify: true
-				},
-				debounceDuration: {
-					type: Number,
-					notify: true,
-					value: 0
 				}
 			},
 			generateRequest: function(){
+				console.log(this.suffix);
+				this.$.ajaxRequest.url = this._buildUrl();
 				this.$.ajaxRequest.generateRequest();
 			},
 			_onResponse: function(r){
+				this.error = null;
+				this.response = r.detail.response;
 				this.fire('response', r);
 			},
-			_onError: function(err){
-				this.fire('error', err);
+			_onError: function(e){
+				this.error = {
```

Comment:
encapsulate this into a behavior too. Maybe we won't use coduno-iron-ajax in some places.